frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_property_edit/mobile_field_type_list.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:appflowy/plugins/database_view/application/field/field_type_option_edit_bloc.dart';
+import 'package:appflowy/plugins/database_view/grid/presentation/widgets/header/field_type_extension.dart';
+
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:appflowy_backend/protobuf/flowy-database2/field_entities.pb.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+typedef SelectFieldCallback = void Function(FieldType);
+
+class MobileFieldTypeList extends StatelessWidget {
+  final FieldTypeOptionEditBloc bloc;
+  final SelectFieldCallback onSelectField;
+  const MobileFieldTypeList({
+    required this.onSelectField,
+    Key? key,
+    required this.bloc,
+  }) : super(key: key);
```

Comment:
Personal preference + super.key constructor
```suggestion
  const MobileFieldTypeList({
    super.key,
    required this.onSelectField,
    required this.bloc,
  });

  final SelectFieldCallback onSelectField;
  final FieldTypeOptionEditBloc bloc;
```